test/features/auth/sign_in_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+feature "Omniauth" do
+  scenario "sign in with github works" do
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.add_mock(:github,
+                            {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.